src/configs/strings/en.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
@cKellyDesign is this generated by hand?

Reply:
yuuuup copied over by hand and given context for translation, it was a pretty interesting exercise actually and helped weed out unused strings!